### PR TITLE
Add manual time support (server-side)

### LINKED ISSUE #13: Support manually adding time
This was an issue that came up during the Subathon stream when the bits subscription stopped working.

Making the request to add time would most likely be done client-side, actually adding time would be done on the server-side (utilizing socket.io).

### PR DESCRIPTION
**Pull request by sh0ckR6**
- Resolves #13

Add support for manual time additions on the server-side

### CODE REVIEW

File: backend/src/index.ts [hunk 1]
```diff
@@ -170,3 +173,15 @@ function addTime(time: number) {
...
+
+
+
+
+
+// -C user inputs time
+// -> addTime
+//   -S addTime()
+//     <- addTime
+//   -C updateTimer()
```
Comment (gtaEPIC) [inline backend/src/index.ts]:
```suggestion
```
monkaW

**Review by gtaEPIC — CHANGES_REQUESTED** [review]

File: backend/src/index.ts (hunk 1 above)
Comment (sh0ckR6) [reply backend/src/index.ts]: Noticed it as you submitted the review, fixed in a6466d0

**Review by gtaEPIC — APPROVED** [review]
Nothing could go wrong right?
Trust

**Comment by sh0ckR6** [comment]
> Nothing could go wrong right? Trust

:D

**Pull request merged**